packages/broken-link-checker/README.md

```diff
@@ -0,0 +1,25 @@
+# Docs Kit Broken Link Checker
```

Comment:
I added the "docs kit" part because what it does is only the customization of the "broken-link-checker" NPM package for the conventions and folder structure of our kit.  If you just need a broken link checker you better directly use the library.

Reply:
I see ok makes sense